<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpreadsheetSystemMainWidget</class>
 <widget class="QWidget" name="SpreadsheetSystemMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spreadsheet System</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="tableNameLabel">
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Table:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="nRowsSpinBox">
     <property name="value">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QSpinBox" name="positionSpinBox">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="previousButton">
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="nRowsLabel">
     <property name="text">
      <string>Rows</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="positionLabel">
     <property name="text">
      <string>Position</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>previousButton</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>nRowsSpinBox</tabstop>
  <tabstop>positionSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
